components/ProductCreation/SelectProducts.js

```diff
@@ -169,16 +184,16 @@ const SelectProducts = ({ select }) => {
         </Button>
       </Icons>
       <ProductTitle>
-        <h2 style={{ marginLeft: "-6px" }}>Shirt</h2>
-        <h2 style={{ marginLeft: "52px" }}>Hat</h2>
-        <h2 style={{ marginLeft: "30px" }}>OuterWear</h2>
-        <h2 style={{ marginLeft: "17px" }}>Shorts</h2>
-        <h2 style={{ marginLeft: "37px" }}>Mug</h2>
-        <h2 style={{ marginLeft: "47px" }}>Bag</h2>
-        <h2 style={{ marginLeft: "53px" }}>Kid</h2>
-        <h2 style={{ marginLeft: "42px" }}>Posters</h2>
+        <h2 style={{ marginLeft: "-.6rem" }}>Shirt</h2>
```

Comment:
Good work calucating the em's this will make sue our product creation is available to every audcaince of every screen size